site/src/styles/global.css

```
@@ -1,49 +1,13 @@
-    U+02DA, U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+20AC, U+2122,
-    U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+/* Self-hosted fonts (Geist, Geist Mono, Inter, JetBrains Mono) are declared
+   via Astro's Fonts API in astro.config.mjs and emitted by the <Font>
+   components in Base.astro: @font-face rules, metric-matched fallbacks, and
+   preload links are generated automatically. display: "optional" keeps CLS at
+   zero. Astro names the generated faces with content hashes, so typography is
+   referenced through the generated CSS variables (--font-inter, etc.), wired
+   into Tailwind's font-sans / font-mono utilities below. */
+@theme {
+  --font-sans: var(--font-inter), ui-sans-serif, system-ui, sans-serif;
+  --font-mono: var(--font-jetbrains-mono), ui-monospace, SFMono-Regular, monospace;
```

Stylelint (17.13.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)